stores/team_store.jsx

```diff
@@ -228,6 +228,31 @@ class TeamStoreClass extends EventEmitter {
         }
     }
 
+    deleteTeam(team) {
```

Comment:
This should be done through redux, these old flux stores should not be getting any new functionality

Reply:
Same feedback

Reply:
@jwilander @enahum Please see changes made. Thanks!